### PR TITLE
Update Dutch translation to 1.15.8

### PR DESCRIPTION
**Pull request by Crementif**

### CODE REVIEW

File: resources/nl/cemu.po [hunk 1]
```diff
@@ -551,116 +551,175 @@ msgstr ""
...
 msgstr "Tekst Kleur"
 
 #: GeneralSettings2.cpp:254
 msgid "Sets the text color of the overlay"
 msgstr "Veranderd de tekst kleur van de overlay"
 
-#: GeneralSettings2.cpp:264
+#: GeneralSettings2.cpp:259 GeneralSettings2.cpp:315
+msgid "Scale"
+msgstr ""
```
Comment (Fs00) [inline resources/nl/cemu.po]: Missing string

File: resources/nl/cemu.po [hunk 2]
```diff
@@ -551,116 +551,175 @@ msgstr ""
...
+
+#: GeneralSettings2.cpp:261
+#, fuzzy
+#| msgid "Sets the text color of the overlay"
+msgid "Sets the scale of the overlay text"
+msgstr "Veranderd de tekst kleur van de overlay"
```
Comment (Fs00) [inline resources/nl/cemu.po]: This hasn't been actually translated, it's the same string as "Sets the text color of the overlay"

File: resources/nl/cemu.po [hunk 3]
```diff
@@ -551,116 +551,175 @@ msgstr ""
...
 msgid "Sets the text color of the overlay"
 msgstr "Veranderd de tekst kleur van de overlay"
 
-#: GeneralSettings2.cpp:264
+#: GeneralSettings2.cpp:259 GeneralSettings2.cpp:315
+msgid "Scale"
+msgstr ""
+
+#: GeneralSettings2.cpp:261
+#, fuzzy
```
Comment (Fs00) [inline resources/nl/cemu.po]:
```suggestion
```

File: resources/nl/cemu.po [hunk 4]
```diff
@@ -551,116 +551,175 @@ msgstr ""
...
+#| msgid "Sets the text color of the overlay"
```
Comment (Fs00) [inline resources/nl/cemu.po]:
```suggestion
```

**Comment by Crementif** [comment]
Everything should work now, at least it did in poedit.

**Comment by Fs00** [comment]
Thanks!

**Pull request merged**